src/components/LoginForm.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import logo from '../assets/logo.png';
+import { TextField, Button } from '@material-ui/core';
+import { makeStyles, createStyles } from '@material-ui/core/styles';
+
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    form: {
+      display: 'flex',
+      flexDirection: 'column',
+      width: '30%',
+      margin: 'auto',
+    },
+    img: {
+      width: '50%',
+    },
+    btn: {
+      backgroundColor: 'teal ',
+      color: 'white',
+      margin: '1.5em auto',
+      textTransform: 'capitalize',
+    },
+    input: {
+      marginTop: '2em',
+      paddingTop: '1em',
+    },
+  })
+);
+const LoginForm: React.FC = () => {
+  const classes = useStyles();
+  return (
+    <div>
+      <form noValidate autoComplete='off' className={classes.form}>
+        <img src={logo} alt='Pockets Logo' className={classes.img} />
+        <TextField id='outlined-basic' label='Your Email' variant='outlined' />
+        <TextField

```

Comment:
Could you change this to a password field? 

Also, there is some alignment issue with the password text field. See screenshot below:
![Screen Shot 2020-10-23 at 1 44 50 PM](https://user-images.githubusercontent.com/10347996/97036567-03bbf900-1536-11eb-8883-593326c842ef.png)